### PR TITLE
Changed isValidLicense Check

### PR DESCRIPTION
**Pull request by irrationalRock**
**Summary**
This PR fixes https://github.com/yarnpkg/yarn/issues/6009.

It no longer throws an error when given `"  "`.

**Motivation**

Yarn would not allow you to install a package if it had ` "  "` in it.

**Test plan**
When running `yarn add mongoose-post-find`, it now runs without producing an error.

### CODE REVIEW

**Comment by Gudahtt** [comment]
Also, this bug was also just fixed in version `3.0.5` of `validate-npm-package-license`. Updating to that version instead would fix the problem as well.

**Comment by irrationalRock** [comment]
Thanks for the advice. I updated the package and it no longer throws the error.

**Comment by Gudahtt** [comment]
Thanks! The last thing that comes to mind is that you updated the `yarn.lock` file to point at `3.0.4`, but not the `package.json` file. That should be updated as well, to ensure it stays above or at that minimum version.

**Comment by irrationalRock** [comment]
Thanks for the help. I updated the yarn.lock and package.json file.

File: package.json [hunk 1]
```diff
@@ -46,7 +46,7 @@
     "tar-stream": "^1.6.1",
     "uuid": "^3.0.1",
     "v8-compile-cache": "^2.0.0",
-    "validate-npm-package-license": "^3.0.3",
+    "validate-npm-package-license": "3.0.4",
```
Comment (Gudahtt) [inline package.json]: That `^` should stay there. There's no reason to pin this dependency.

**Comment by irrationalRock** [comment]
Thanks for the suggestion. I changed it to `^`.

**Comment by arcanis** [comment]
Thanks 💯

**Pull request merged**